### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [jrmiller89/prj-rev-bwfs-dasmoto#1]
Don't forget to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute)!

The attribute helps optimize your site for search engine results, increase the [accessibility](https://www.codecademy.com/articles/accessibility) of your site, and it is used as a backup in the chance that your image cannot be rendered. 

https://github.com/jrmiller89/prj-rev-bwfs-dasmoto/blob/e367f56350aaff09bb4aa9fb8d5ffb02c83337fd/Dasmoto's%20Arts%20%26%20Crafts_Justin%20Miller/index.html#L11

**Comment by jrmiller89** [comment]
Ah totally forgot about this! Thank you.
